### DISCUSSION TITLE
Refinement of survey responses - planning cycle

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#706] [General]
We are currently in the process of analysing your responses to the [Pipeline usage survey](https://github.com/daisy/pipeline/discussions/682) held earlier this year. Some of the feedback requires some more elaboration or discussion. Below are some responses in the topic "planning cycle":

---
- **Q**: Does the DAISY Pipeline planning cycle once-a-year suit your organization's requirements?
  - Two organisations indicated that they would prefer a quarterly or even monthly planning cycle.
---

I would like to know:

- Are you happy with the new [time based release schedule](https://github.com/daisy/pipeline/discussions/684)?
- The high-level operational plan still happens only once a year, but thanks to the time based release schedule and the improved communication, work items that can not be planned at the beginning of the year, such as newly discovered critical bugs or new high-priority or easy to fix feature requests, can be part of a quarter-yearly planning cycle. Is this a good solution?

**Comment by josteinaj** [comment]
I think this is a good solution. I'm happy :)

**Comment by torchtrust** [comment]
I agree
